src/main/resources/META-INF/rewrite/mockito.yml

```diff
@@ -174,6 +174,7 @@ recipeList:
       version: 3.x
       onlyIfUsing: org.mockito.junit.jupiter.*
       acceptTransitive: true
+      scope: test
```

Comment:
Hmm; I thought in the past we intentionally did not specify a scope, as `AddDependency` should then determine the scope based on where the `onlyIfUsing` classes are being used. It's not per-se wrong, but could for a shared testing library then for instance not add the dependency in the correct scope for downstream users of that testing lib.

How'd you arrive at this change? And did you verify it against our internal services that this indeed fixes it?

Reply:
The behavior of determining the right scope is not documented. It says default is `runtime` and we saw this recipe add the dependency as `implementation`, updates it up to modern Spring and also updating the `testImplementation`.